pipelines/internal/commands/watch/watch.go

```diff
@@ -55,6 +55,9 @@ func Invoke(ctx context.Context, service *genomics.Service, project string, argu
 
 func watch(ctx context.Context, service *genomics.Service, name string) (interface{}, error) {
 	var events []*genomics.Event
+	const minDelay = 5 * time.Second
```

Comment:
You never need 'minDelay' again... What about something like

delay := 5 * time.Second
...
time.Sleep(delay)
delay *= 1.5
...
if limit := time.Minute; delay > limit {
  delay = limit
}

Reply:
I do use minDelay once more - to reset the delay after each even

Reply:
OK so same basic changes but we can have const initialDelay and assign that originally

Reply:
ok I changed the name and removed the maxDelay from const (limit var in the if loop instead)